bitbots_head_behavior/src/bitbots_head_behavior/actions/look_at.py

```diff
@@ -38,13 +38,16 @@ def get_motor_goals_from_point(self, point):
         states = response.solution.joint_state
         return states.position[states.name.index('HeadPan')], states.position[states.name.index('HeadTilt')]
 
-    def _look_at(self, point):
+    def _look_at(self, point, min_pan_delta=0, min_tilt_delta=0):
```

Comment:
I like the idea of deltas* to being overwritable but would it not make sense to have the config values as default instead?

```suggestion
    def _look_at(self, point, min_pan_delta=self.blackboard.config['ball_tracking_min_pan_delta'], min_tilt_delta=self.blackboard.config['ball_tracking_min_tilt_delta']):
```